pyproject.toml

```diff
@@ -23,25 +22,31 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
+python = ">=3.9.2,<3.13"
```

Comment:
Do we still need to support Python 3.8 for Nautobot 1.6.0-2.3.16?

Reply:
IIRC pylint 3.x dropped 3.8 support so it won't matter.